### DISCUSSION TITLE
Switching Branches

### DISCUSSION THREAD

**Discussion by lissakariuki** [community/community#63482] [New to GitHub]
Hey guys,
I have a quick question: My default branch name is master, but i saw that a lot of people are calling their default branch main, which i would prefer. How can i change it to be main out of the box ?

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
